composables/useSubscriptionGraphql.ts

```diff
@@ -32,16 +32,7 @@ export default function ({
 
   async function pollData() {
     try {
-      const response = await $fetch(httpUrl, {
-        method: 'POST',
-        body: {
-          query: `
-          query {
-            ${query}
-          }
-        `,
-        },
-      })
+      const response = await fetchGraphql(query, { client })
```

Comment:
As the only use case, Is it necessary to move to the utils function?

Reply:
a prev version of the code was using it , removed it since it's not used anymore